twisted/web/test/test_http2.py

```diff
@@ -1403,6 +1403,32 @@ def validate(streamID):
         return a._streamCleanupCallbacks[1].addCallback(validate)
 
 
+    def test_resetAfterBody(self):
+        """
+        A client that immediately resets after sending the body causes Twisted
+        to send no response.
+        """
+        f = FrameFactory()
```

Comment:
The frame factory is a state container which includes, amongst other things, header compression state. This means that we can only use one in a given test.